docs/releases/1.11.txt

```diff
@@ -325,6 +325,12 @@ URLs
 
 * ...
 
+Utilities
+~~~~~~~~~
+
+* The new :func:`django.utils.timezone.localdate` function returns a
+  :class:`~datetime.datetime`'s date in the current time zone.
```

Comment:
You would need to use a right quote here to prevent Sphinx from rendering it in the opposite direction.